lib/Map/LeafletVisualizer.js

```diff
@@ -722,8 +722,8 @@ LeafletVisualizer.prototype.visualizersCallback = function(leafletScene, dataSou
     return [new LeafletGeomVisualizer(leafletScene, entities)];
 };
 
-function featureClicked(visualizer, entity) {
-    visualizer._leafletScene.featureClicked.raiseEvent(entity);
+function featureClicked(visualizer, entity, e) {
```

Comment:
Can you be more specific than `e`? What sort of object is this? (Here and in `featurePicked`.)